src/main/java/at/pfeifer/chatapp/controller/ChatController.java

```diff
@@ -30,12 +44,30 @@ public class ChatController implements Initializable {
     @FXML
     private TextField signatureInput;
 
+    @FXML
+    void saveMessage() {
+        var primaryStage = new Stage();
+        var vBox = new VBox();
+        primaryStage.setScene(new Scene(vBox, 0, 0));
+        primaryStage.setTitle("dummy scene to launch file explorer");
+        var fileChooser = new FileChooser();
+
+        FileChooser.ExtensionFilter extFilter = new FileChooser.ExtensionFilter("TXT files (*.txt)", "*.txt");
+        fileChooser.getExtensionFilters().add(extFilter);
+
+        var file = fileChooser.showSaveDialog(primaryStage);
```

Comment:
This could be simplified
```suggestion
        var fileChooser = new FileChooser();
        fileChooser.getExtensionFilters().add(
                new FileChooser.ExtensionFilter("TXT files (*.txt)", "*.txt")
        );
        var file = fileChooser.showSaveDialog(new Stage());
```